src/net.rs

```diff
@@ -107,6 +107,7 @@ pub struct TcpNetEntry {
 pub struct UdpNetEntry {
     pub local_address: SocketAddr,
     pub remote_address: SocketAddr,
+    pub state: TcpState,
```

Comment:
I think I would prefer to have a new `UdpState` enum for this (since a UDP socket can't be in all of the same states that a TCP socket can)

Reply:
(I realize that internally the kernel uses the same state type values as TCP connections, but having a new `UdpState` would reduce confusion I think)

Reply:
You mean something like this?

```rust
enum UdpState {
    Established = 1,
    Close = 7,
}
```

Reply:
Yeah, exactly

Reply:
See 8baacbe.

Reply:
See 8a55ab4f0cac1d36077ba88149441d7b745bc222.